integration/test_flow_integration.py

```diff
@@ -320,7 +343,7 @@ def test_join_by_key(setup_kv_teardown_test):
 
 
 def test_join_by_key_specific_attributes(setup_kv_teardown_test):
-    table = Table(setup_kv_teardown_test, V3ioDriver())
+    table = Table(setup_kv_teardown_test["table_name"], get_driver(setup_kv_teardown_test))
```

Comment:
I think it would be nicer to have `setup_kv_teardown_test` return a tuple?
As in
```suggestion
    table_name, _ = setup_kv_teardown_test
```